tests/supertest/requestSender.spec.ts

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import express from 'express';
+import { expectTypeOf } from 'expect-type';
+import bodyParser from 'body-parser';
+import { RequestSender } from '../../src/requestSender/types';
+import { createRequestSender } from '../../src/requestSender/requestSender';
+import { paths, operations } from './types';
+
+describe('requestSender', () => {
+  let expressApp: express.Application;
+  let requestSender: RequestSender<paths, operations>;
+
+  beforeEach(async function () {
+    expressApp = express();
+    expressApp.use(bodyParser.json());
+    requestSender = await createRequestSender<paths, operations>('tests/supertest/openapi3.yaml', expressApp);
+  });
+
+  describe('operations', () => {
+    it('should send a simple request with no parameters', async () => {
+      expect.assertions(4);
+      expressApp.get('/simple-request', (req, res) => {
+        expect(req.query).toEqual({});
+        // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Unnecessary as there is already this comment at the first of the file. Choose one way

Reply:
removed